### ISSUE TITLE
how to run multiple test cases in an order and get result

### ISSUE THREAD

**Issue by renas** [RobotiumTech/robotium#223]
_From [Tashfeen...@gamevstudios.com](https://code.google.com/u/114838328977690241533/) on August 28, 2011 00:53:24_

how to run multiple test cases in an order and get result

_Original issue: http://code.google.com/p/robotium/issues/detail?id=144_

**Comment by renas** [comment]
_From [renasr...@gmail.com](https://code.google.com/u/108052454046732184536/) on August 28, 2011 22:51:39_

Please download the example project and see how it is designed. There the test cases are run in order.

**Status:** Invalid

**Comment by renas** [comment]
_From [tomaz.bu...@gmail.com](https://code.google.com/u/104008391460745889736/) on August 29, 2011 06:35:50_

I'm having same issue.

In example given (NotePadTest) they run in alphabeticly order (A - > Z) and this is why example is in right order. How can we set order if we want to use custom order?

I have four tests:
-testOne
-testTwo
-testTheree
-testFour

If I run this first testFour will be run and later others.

Another question. Do I have to use "test" in every test function to work? I tried custom names and didn't work.

Thanks.
